src/System.Text.RegularExpressions/ref/System.Text.RegularExpressions.cs

```diff
@@ -221,7 +224,7 @@ public partial class RegexMatchTimeoutException : System.TimeoutException, Syste
         public string Input { get { throw null; } }
         public System.TimeSpan MatchTimeout { get { throw null; } }
         public string Pattern { get { throw null; } }
-        void System.Runtime.Serialization.ISerializable.GetObjectData(System.Runtime.Serialization.SerializationInfo si, System.Runtime.Serialization.StreamingContext context) { }
+        void System.Runtime.Serialization.ISerializable.GetObjectData(System.Runtime.Serialization.SerializationInfo info, System.Runtime.Serialization.StreamingContext context) { }
```

Comment:
This might break reflection scenarios right? @ericstj

Reply:
This was changed in b5105340f57334b553e204e90df9358e4e1fed72. Looks like the ref source hasn't been updated in a while.

Reply:
@stephentoub what would be the best course of action here? The change has been in the runtime for a while, so probably makes sense to update netstandard too?

Reply:
I think it's ok to change.  While technically it could break someone, I really doubt it will in any meaningful way.